A61-IPv4-IPv6-dualstack-backends.md

```diff
@@ -545,6 +545,7 @@ for (i = 0; i < ring.size(); ++i) {
     return PICK_QUEUE;
   }
 }
+return PICK_FAIL;
```

Comment:
I took the liberty to add this clarification to A62 -- handle the case of transient failure. Although I presume that the aggregated state reported to the parent may prevent the case from ever happening?

Reply:
C-core does essentially the same, attaching additional info: https://github.com/grpc/grpc/blob/master/src/core/load_balancing/ring_hash/ring_hash.cc#L356-L358